shared/pages/Wallet/Row/Row.js

```diff
@@ -80,7 +116,22 @@ export default class Row extends Component {
       isBalanceFetching: false,
     }))
   }
-
+  
+  shouldComponentUpdate(nextProps, nextState) {
+    const getComparableProps = (props) => {
```

Comment:
```suggestion
    const getComparableProps = ({ item }) => {
```

можно вот так делать, мб так будет компактнее?